src/components/BlockQuote/BlockQuoteParagraph.js

```diff
@@ -25,11 +25,12 @@ const styles = {
   })
 }
 
-const BlockQuoteParagraph = ({ children, attributes }) => {
+const BlockQuoteParagraph = ({ children, attributes, style }) => {
```

Comment:
same as above: write a comment specific block quote paragraph in `components/CommentBody/web/Blockquote.js`. Overwriting media query styles is too hard to debug. DRY benefit is small here.